src/crimson/osd/main.cc

```diff
@@ -79,26 +122,36 @@ int main(int argc, char* argv[])
   }
   std::string cluster_name;
   std::string conf_file_list;
+
```

Comment:
please avoid adding empty lines. see https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
will take a look, we can put it to low priority to do so.